src/utils/postFunctions.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+
+function postArticleComment(articleId,author,body){
+
+    return axios.post(`https://nc-news-ldv7.onrender.com/api/articles/${articleId}/comments`, {
+        "username": author,
+        "body": body
+        })
+     .then((response)=>{
+        return response
+     })
```

Comment:
We should only have one catch block in any promise chain (at the end). In this case the catch block should be in the file in which postArticleComment was invoked